cardsWorthTen.js

```diff
@@ -21,7 +72,25 @@ const cards = [
  * @param {array} cards
  * @return {string} displayVal
  */
-const cardsWorthTen = cards => {};
+const cardsWorthTen = cards => {
+
+  let filteredCards = cards.filter((card) => card.val === 10);
+  let filteredVal = [];
+
+  for (let i = 0; i < filteredCards.length; i++) {
+
+    filteredVal = console.log(`${filteredCards[i].displayVal}`);
+
+  };
+
+  return (filteredVal)
+};
+
+
+
+cardsWorthTen(cards);
```

Comment:
I see you removed the console log that was provided by the boiler plate, and that is adding to the confusion over `console.log()`  You want to keep that as is, because the mental model is more accurately like "the function _returns_ a value, but the console log here simply writes the value that comes from calling the function to output so that we can see it"

In general, `console.log()` is a handy tool for developers, but really does not play a role in actual programming, other than giving you a way to peek at values when you need to. For most production code you will delete all of the console.log() statements before shipping. It's just a helper.